dephell/actions/_contributing.py

```diff
@@ -0,0 +1,95 @@
+from typing import Any, Dict, Optional
+
+CONTRIBUTING = """
+# Contributing to dephell
```

Comment:
Let's replace it everywhere in template by `{{ project name }}` and pass just a current dir name for the beginning.